File: Samples/ProgrammingLanguagePopularity/LanguagePopularity.vb
Imports Ezfx.Csv

<CsvObject(CodePage:=936, Description:="", HasHeader:=True, MappingType:=CsvMappingType.Order, Name:="")> _
Public Class LanguagePopularity
    ''' <summary>
    ''' 0, Position
    ''' </summary>
    <SystemCsvColumn(Alias:="", Ordinal:=0, Name:="Position")>
    Public Property Position() As String
    ''' <summary>
    ''' 1, ProgrammingLanguage
    ''' </summary>
    <SystemCsvColumn(Alias:="", Ordinal:=1, Name:="ProgrammingLanguage")>
    Public Property ProgrammingLanguage() As String
    ''' <summary>
    ''' 2, Ratings
    ''' </summary>
    <SystemCsvColumn(Alias:="", Ordinal:=2, Name:="Ratings")>
    Public Property Ratings() As String
    ''' <summary>
    ''' 3, Delta
    ''' </summary>
    <SystemCsvColumn(Alias:="", Ordinal:=3, Name:="Delta")>
    Public Property Delta() As String
    ''' <summary>
    ''' 4, Status
    ''' </summary>
    <SystemCsvColumn(Alias:="", Ordinal:=4, Name:="Status")>
    Public Property Status() As String

End Class

File: Samples/Languages/LanguagePopularity.cs
using Ezfx.Csv;
using System;

namespace Languages
{

    [CsvObject(CodePage = 936, Description = "", HasHeader = true, Name = "")]
    public class LanguagePopularity
    {
        /// <summary>
        /// 0, Position
        /// </summary>
        [SystemCsvColumn(Name = "Position", Ordinal = 0, Alias = "")]
        public int Position { get; set; }

        /// <summary>
        /// 1, Programming Language
        /// </summary>
        [SystemCsvColumn(Name = "Programming Language", Ordinal = 1, Alias = "")]
        public string ProgrammingLanguage { get; set; }

        /// <summary>
        /// 2, Ratings
        /// </summary>
        [SystemCsvColumn(Name = "Ratings", Ordinal = 2, Alias = "")]
        public float Ratings { get; set; }

        /// <summary>
        /// 3, Delta
        /// </summary>
        [SystemCsvColumn(Name = "Delta", Ordinal = 3, Alias = "")]
        public Nullable<float> Delta { get; set; }

        /// <summary>
        /// 4, Status
        /// </summary>
        [SystemCsvColumn(Name = "Status", Ordinal = 4, Alias = "")]
        public string Status { get; set; }

    }
}
